### ISSUE TITLE
[Victory] Fwd and Bwd should start by choosing an input model

### ISSUE THREAD

**Issue by Surbhi2111** [eMoflon/emoflon-ibex-ui#167]
The FWD and BWD transformations should start by prompting the user to choose an input model and there shouldn't be a "default" model to start these transformations.

**Comment by Ayurshi** [comment]
yes makes a lot of sense. Also to add to it, currently saved models with time stamp can't be used as src/trg to run the fwd/bwd transformation, only the models named as src.xmi/trg.xmi can we used.
